collections/vegetation-phenology-and-productivity-parameters-season-1/README.MD

```diff
@@ -88,20 +88,10 @@ The available bands include: 13 parameters and QFLAG
 ### Representative Images 
 
 <img src="sosv-candasnos-spain-2019.PNG" alt="Vegetation index value at day of start of season 1" width="100%"/>
-<br>
-
```

Comment:
You need to leave 1 empty line before the caption otherwise the italic is not detected an the caption is having the stars around it. You can always take a look at the changes already via "view this file".

![2022_01_03_12_31_29_Window](https://user-images.githubusercontent.com/60705209/147925870-683fe25a-84e9-4a62-a02b-8170866af643.png)

Reply:
thanks ..  I was getting confused why the italics are not beig detected

Reply:
Better now though with the removed breaks also between images it is now hard to see to which image the caption belongs. I'd leave the 2 breaks between the images to make this clear.

Reply:
Looks okay now.